uiowa/uiowa
# Move the headline field to outside of the slider container.

The slider component in UIDS has no explicit headline field defined, so the headline field attached to it in this project should be displayed outside the `<div class="slider">` container element. This should hopefully prevent unexpected behavior since we don't test for this type of condition in UIDS.

# How to test
* `blt ds --site=grad.uiowa.edu`
* `drush @grad.local uli /`
* Inspect the HTML of the slider element on the page that is opened up and observe that the headline element is rendered outside the slider.
